rebar.config

```diff
@@ -110,7 +114,10 @@
                    {template, "rel/files/vm.dist.args",     "etc/vm.dist.args"},
 
                    {copy, "priv/logo.txt", "priv/logo.txt"},
-                   {copy, "VERSION", "priv/VERSION"}
+                   {copy, "VERSION", "priv/VERSION"},
+
+                   %% cowboy_swagger for generating documentation
+                   {copy, "./_build/default/lib/cowboy_swagger/priv/swagger", "priv/swagger"}
```

Comment:
I wonder if we need this at all. Maybe function `code:priv_dir(cowboy_swagger)` could be used and change the `static_files` parameter of `cowboy_swagger` application when the REST API listener starts? What do you think?

Reply:
You see, I use approach from example of `swagger_cowboy` - https://github.com/inaka/cowboy_swagger/blob/master/example/rebar.config#L35 and I not sure if we can apply other approach

Reply:
Done.